regulament_pregatit_feedback.txt

```diff
@@ -660,6 +665,79 @@ Art. 106.
+
+Art. 107+2. În grupurile de Cercetași, una din unități (sau unitatea, în cazul în care exită doar una) poate fi, prin excepție, formată dintr-o singură patrulă. Liderul care îndrumă unitatea trebuie să fie lider ONCR, dar poate să nu facă parte din Grupul de Cercetași; este ales de către membrii unității, în baza planului de mentorat și a propunerilor făcute de structura mentor.
+
+Art. 107+3. Grupurile de Cercetași există exclusiv sub mentoratul și gestiunea administrativă a unei structuri mentor, care de regulă este un Centru Local care îndeplinește condițiile minime pentru această situație. 
+
+Art. 107+4. În situații excepționale, în care nu este posibilă arondarea la un Centru Local din proximitatea geografică, Consiliul Director poate împuternici orice alt Centru Local eligibil să mentoreze grupul de cercetași, cu acordul acelui Centru Local. Alternativ, Consiliul Director poate împuternici nominal un grup de minim 3 lideri din ONCR care să asigure mentoratul grupului de cercetași, acesta primind formal mentoratul Consiliului Director; într-o astfel de situație excepțională, liderii și membrii grupului de cercetași vor participa la activități cercetășești și administrative în Centrele Locale ale liderilor desemnați ca mentori.
+
+Art. 107.5. Pentru înființarea unui Grup de Cercetași sunt necesare:
+	(1) Acordul unui grup informal de persoane de a forma împreună un Grup de cercetași
+	(2) Elaborarea, de către o structură mentor alături de Grupul de cercetași a unui Plan de Mentorat pe o perioadă de 1 an, care să descrie în ce condiții se va desfășura mentoratul și care conține minim:
+		(a) angajamentul reciproc al Grupului de Cercetași și al structurii mentor pentru dezvoltarea cercetășiei în Grupul de Cercetași, 
+		(b) obiective și un plan de acțiune concret cu privire la participarea Grupului de cercetași la activități cercetășești și organizatorice ale structurii mentor, precum și activitățile proprii Grupului de cercetași
+		(c) termenele pentru planificarea și evaluarea planului de mentorat
+		(d) persoanele desemnate din partea Grupului de cercetași și a stucturii mentor pentru urmărirea implementării planului de mentorat
+	(3) Acordul unei structuri de a prelua mentoratul Grupului de cercetași, în baza Planului de mentorat de la punctul (2), în condițiile art. 107+3 și 107+4; în situația din art. 107+3, acordul este dat printr-o hotărâre de Adunare Generală; în situația art. 107.4, acordul este dat printr-o decizie a Consiliului Director.
+	(4) Adițional, în cazul în care structura mentor este un Centru Local, este necesar și avizul Consiliului Director pentru Planul de mentorat.
+
+Art. 107.5. Planul de mentorat poate fi reînnoit după perioada de 1 an și în situații excepționale poate fi modificat în timpul implementării, cu acordul tuturor părților implicate, exprimat conform Art. 107.5, punctele (3) și (4).
+Art. 107.5+0. Cerințele minime pentru Planul de Mentorat sunt elemente de procedură internă a ONCR.
```

Comment:
Mă confuzează puțin numerotarea (107.5+0). 5 e alineat sau un nou articol? Știu că nu e chiar relevant în momentul ăsta dar mă ajută să înțeleg ca să urmăresc mai ușor flow-ul

Reply:
Articol nou